src/components/Song/Upvote.js

```diff
@@ -1,26 +1,23 @@
 import { contains } from 'ramda'
 import React from 'react'
-import Arrow from '../../img/arrow-up.svg'
 import { readVotesFromStorage } from 'util/votes'
 
-const isUpvoted = songId => {
-  let persistedVotes = readVotesFromStorage()
-  return contains(songId, persistedVotes)
-}
-
 const Upvote = ({changeVote, songId}) => {
-  if (isUpvoted(songId)) return <div>VOTED!</div>
+  const isUpvoted = contains(songId, readVotesFromStorage())
```

Comment:
this logic also happens in `util/votes` - we could pull it out into another exported function and save the import of `contains` in this module

Reply:
Nice